### PR TITLE
Add Analysis Export

### PR DESCRIPTION
**Pull request by barshathakuri**
- Addresses #2466 
- Depends on the https://github.com/the-deep/server/pull/1095

## Changes

* Detailed list or prose of changes
* Breaking changes
* Changes to configurations

## This PR doesn't introduce any:

- [x] temporary files, auto-generated files or secret keys
- [x] build works
- [x] eslint issues
- [x] typescript issues
- [x] codegen errors
- [x] `console.log` meant for debugging
- [x] typos
- [x] unwanted comments
- [x] conflict markers

## This PR contains valid:

- [x] permission checks
- [x] translations

### CODE REVIEW

**Review by AdityaKhatri — DISMISSED** [review]

File: app/views/Project/AnalysisModule/Analysis/index.tsx [hunk 1]
```diff
@@ -203,6 +211,16 @@ function Analysis(props: ComponentProps) {
                     >
                         <IoCopyOutline />
                     </QuickActionButton>
+                    <QuickActionButton
+                        name="clone"
+                        onClick={hideCreateNewExportModal}
+                        disabled={disabled}
+                        title={_ts('analysis', 'cloneAnalysisButtonTitle')}
```
Comment (AdityaKhatri) [inline app/views/Project/AnalysisModule/Analysis/index.tsx]: Change title to 'Export'

File: app/views/Project/AnalysisModule/Analysis/index.tsx [hunk 2]
```diff
@@ -203,6 +211,16 @@ function Analysis(props: ComponentProps) {
                     >
                         <IoCopyOutline />
                     </QuickActionButton>
+                    <QuickActionButton
+                        name="clone"
```
Comment (AdityaKhatri) [inline app/views/Project/AnalysisModule/Analysis/index.tsx]:
```suggestion
                        name={undefined}
```

File: app/views/Project/AnalysisModule/Analysis/index.tsx [hunk 3]
```diff
@@ -203,6 +211,16 @@ function Analysis(props: ComponentProps) {
...
+                        onClick={hideCreateNewExportModal}
```
Comment (AdityaKhatri) [inline app/views/Project/AnalysisModule/Analysis/index.tsx]: handle this properly

File: app/views/Project/AnalysisModule/Export/index.tsx [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+        newAssessmentModalShown,
+        showNewAssessmentModal,
+        hideNewAssessmentModal,
+    ] = useModalState(false);
+
+    return (
+        <Tabs
+            onChange={setActiveTab}
+            value={activeTab}
+        >
```
Comment (AdityaKhatri) [inline app/views/Project/AnalysisModule/Export/index.tsx]: Tabs is not required.

File: app/views/Project/AnalysisModule/Export/index.tsx [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+
+type ExportType = 'export-entry-history' | 'export-assessment-history';
+
+const entryType: ExportDataTypeEnum[] = ['ENTRIES'];
+const assessmentType: ExportDataTypeEnum[] = ['PLANNED_ASSESSMENTS', 'ASSESSMENTS'];
+
+function Export() {
+    const { project } = React.useContext(ProjectContext);
+    const activeProject = project ? project.id : undefined;
+    const [activeTab, setActiveTab] = useState<ExportType | undefined>('export-entry-history');
```
Comment (AdityaKhatri) [inline app/views/Project/AnalysisModule/Export/index.tsx]: Tabs is not required.

**Review by AdityaKhatri — CHANGES_REQUESTED** [review]

**Comment by AdityaKhatri** [comment]
@barshathakuri , check for linting and typecheck issues.
Fix deepscan issues.

File: app/Base/configs/routes.tsx [hunk 6]
```diff
@@ -226,7 +227,7 @@ const entryEditRoute = wrap({
 });
 const analysis = wrap({
     parent: { path: projectRoute.path },
-    path: '/analysis/',
+    path: '/analysis-module/',
```
Comment (samshara) [inline app/Base/configs/routes.tsx:230]: have we decided to use this path?

**Review by AdityaKhatri — APPROVED** [review]

**Pull request merged**